src/flint/routers/helpers.py

```diff
@@ -24,3 +30,21 @@ def log_telemetry(
     row.update(properties)
     telemetry.append(row)
 
+
+def ogg2mp3(audio_url, user_id):
+    # Get the response of the OGG file
+    response = requests.get(audio_url)
+    # Get the redirect URL result
+    url = response.url # `url` value something like this: "https://s3-external-1.amazonaws.com/media.twiliocdn.com/<some-hash>/<some-other-hash>"
+    # Set file path
+    filepath = f"/tmp/{user_id}_audio"
```

Comment:
If a user sends two voice messages in quick succession, this could hypothetically overwrite the existing file that's being transcribed/processed?

Reply:
We could use a hashed value based on the timestamp to avoid the collision issue.